### DISCUSSION TITLE
Can I get a permanent link for each photo?

### DISCUSSION THREAD

**Discussion by fyzzy1943** [thumbsup/thumbsup#303] [Q&A]
Now only each album has a link

**Comment by rprieto** [comment]
Hi, this is not available out of the box, but LightGallery does support it using the `hash` plugin. That plugin is included in the default themes.

 You can try customizing the theme (`album.hbs`) to add `hash: true` in the options:

```js
$("#media").lightGallery({
...
})
```

If that works well we could make it the default on all themes!
